src/components/pages/Flagging/FlagFeed.tsx

```diff
@@ -0,0 +1,210 @@
+            <Text style={sharedStyles.infoHeader}>
+              There are no remaining flags on this account!
+            </Text>
+          </View>
+        )}
+        renderItem={dataItem => {
+          const { flaggingMember, flagOperation } = dataItem.item;
+          return (
+            <View style={styles.listItem}>
+              <View style={listItemHeaderStyle}>
+                <MemberThumbnail
+                  member={flaggingMember}
+                  style={styles.listItemHeaderThumbnail}
+                />
+                <View style={listItemHeaderTextStyle}>
+                  <Text>
+                    <MemberName member={flaggingMember} /> flagged this account.
+                  </Text>
+                  <Text style={listItemHeaderTextTimestampStyle}>
+                    {formatRelative(
+                      flagOperation.created_at,
+                      new Date()
+                    ).toUpperCase()}
+                  </Text>
+                </View>
+              </View>
+              <Text style={styles.listItemBody}>
+                {flagOperation.data.reason}
+              </Text>
+              <View style={styles.listItemActions}>
+                <Button
+                  title="Resolve flag"
+                  style={styles.listItemButton}
+                  textStyle={styles.listItemButtonText}
+                  onPress={() =>
+                    navigation.navigate(RouteName.ResolveFlagMemberPage, {
+                      flagToResolveOperation: flagOperation,
+                      apiCallId: generateRandomIdentifier()
+                    })
+                  }
+                />
+              </View>
+            </View>
+          );
+        }}
+      />
+    </View>
+  );
+};
+
+const mapStateToProps: MapStateToProps<StateProps, OwnProps, RahaState> = (
+  state,
+  ownProps
+) => {
+  const member = ownProps.navigation.getParam("member");
+  if (!member) {
+    throw new Error("No member passed to FlagFeed page.");
+  }
+  // Refresh member state
+  const refreshedMember = getMemberById(
+    state,
+    member.get("memberId")
+  ) as Member;
+  const flagOperationIds = refreshedMember.get("operationsFlaggingThisMember");
+  return {
+    flagData: (state.operations.filter(op =>
+      flagOperationIds.includes(op.id)
+    ) as List<FlagMemberOperation>).map(flagMemberOperation => {
+      const flaggingMember = getMemberById(
+        state,
+        flagMemberOperation.creator_uid
+      );
+      if (!flaggingMember) {
+        throw new Error(
+          `Invalid flag operation with id: ${
+            flagMemberOperation.id
+          } in flag feed. Flag creator could not be found.`
+        );
+      }
+      return {
+        flaggingMember,
+        flagOperation: flagMemberOperation
+      };
+    })
+  };
+};
+
+export const FlagFeedPage = connect(mapStateToProps)(FlagFeedPageView);
+
+const LEFT_MARGIN = 50;
+
+const listStyle: ViewStyle = {
```

Comment:
(Not related to your PR, but why do we do this instead of putting it directly in styles? Feels like unnecessary boilerplate..)